src/System.ComponentModel.TypeConverter/src/System/ComponentModel/AttributeCollection.cs

```diff
@@ -5,7 +5,6 @@
 using System.Reflection;
 using System.Collections;
 
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Security", "CA2112:SecuredTypesShouldNotExposeFields", Scope = "type", Target = "System.ComponentModel.AttributeCollection")]
```

Comment:
The tooling we use for security signoffs doesn't know that we don't use CAS. Suppressions help us weed out the noise.